glide-core/src/request_type.rs

```diff
@@ -329,33 +743,34 @@ impl RequestType {
             RequestType::XGroupCreate => Some(get_two_word_command("XGROUP", "CREATE")),
             RequestType::XGroupDestroy => Some(get_two_word_command("XGROUP", "DESTROY")),
             RequestType::XTrim => Some(cmd("XTRIM")),
-            RequestType::HSetNX => Some(cmd("HSETNX")),
+            RequestType::HSetNx => Some(cmd("HSETNX")),
             RequestType::SIsMember => Some(cmd("SISMEMBER")),
-            RequestType::Hvals => Some(cmd("HVALS")),
-            RequestType::PTTL => Some(cmd("PTTL")),
+            RequestType::HVals => Some(cmd("HVALS")),
+            RequestType::PTtl => Some(cmd("PTTL")),
             RequestType::ZRemRangeByRank => Some(cmd("ZREMRANGEBYRANK")),
             RequestType::Persist => Some(cmd("PERSIST")),
             RequestType::ZRemRangeByScore => Some(cmd("ZREMRANGEBYSCORE")),
             RequestType::Time => Some(cmd("TIME")),
-            RequestType::Zrank => Some(cmd("ZRANK")),
+            RequestType::ZRank => Some(cmd("ZRANK")),
             RequestType::Rename => Some(cmd("RENAME")),
-            RequestType::DBSize => Some(cmd("DBSIZE")),
-            RequestType::Brpop => Some(cmd("BRPOP")),
-            RequestType::Hkeys => Some(cmd("HKEYS")),
+            RequestType::DbSize => Some(cmd("DBSIZE")),
+            RequestType::BRPop => Some(cmd("BRPOP")),
+            RequestType::HKeys => Some(cmd("HKEYS")),
             RequestType::PfAdd => Some(cmd("PFADD")),
             RequestType::PfCount => Some(cmd("PFCOUNT")),
             RequestType::PfMerge => Some(cmd("PFMERGE")),
             RequestType::RPushX => Some(cmd("RPUSHX")),
             RequestType::LPushX => Some(cmd("LPUSHX")),
-            RequestType::Blpop => Some(cmd("BLPOP")),
+            RequestType::BLPop => Some(cmd("BLPOP")),
             RequestType::LInsert => Some(cmd("LINSERT")),
-            RequestType::Spop => Some(cmd("SPOP")),
+            RequestType::SPop => Some(cmd("SPOP")),
             RequestType::ZMScore => Some(cmd("ZMSCORE")),
             RequestType::ZDiff => Some(cmd("ZDIFF")),
             RequestType::ZDiffStore => Some(cmd("ZDIFFSTORE")),
             RequestType::SetRange => Some(cmd("SETRANGE")),
             RequestType::ZRemRangeByLex => Some(cmd("ZREMRANGEBYLEX")),
             RequestType::ZLexCount => Some(cmd("ZLEXCOUNT")),
+            _ => todo!(),
```

Comment:
Are there actually more commands we're missing here? Or is this just a catch-all case for future added commands?